tests/Feature/WebRouteTest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Tests\Feature;
+
+use Tests\TestCase;
+
+class WebRouteTest extends TestCase
+{
+    // phpcs:disable PSR1.Methods.CamelCapsMethodName.NotCamelCaps
+
+    /**
+     * Test the /auth/token route
+     *
+     *  @return void
+     */
+    // phpcs:ignore
+    public function test_welcome_route()
+    {
+        $response = $this->get('/');
+
+        $response->assertStatus(200);
+        $response->assertViewIs('welcome');
```

Comment:
This is so magnificently simple, I love it :)

Reply:
It's magic :sparkler: :sparkler: